CHANGELOG.md

```diff
@@ -65,7 +65,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated `trainer.running_sanity_check` in favor of `trainer.sanity_checking` ([#4945](https://github.com/PyTorchLightning/pytorch-lightning/pull/4945))
 
 
-- Deprecated metrics in favor of `torchmetrics` ([#6505](https://github.com/PyTorchLightning/pytorch-lightning/pull/6505))
+- Deprecated metrics in favor of `torchmetrics` ([#6505](https://github.com/PyTorchLightning/pytorch-lightning/pull/6505),
+
+
+    [#6515](https://github.com/PyTorchLightning/pytorch-lightning/pull/6515),
+    
+    )
```

Comment:
this doesn't look right

Reply:
it is just spacing around to avoid collisions

Reply:
But space is needed between bullet points (`-`), not within a bulletpoint

Reply:
ok, I'll check it with the next rebase as #6530 is required